packages/core/src/constants/type/Config.ts

```diff
@@ -88,6 +93,10 @@ export type FullConfig = {
    * Refresh standard calls each time the n-th block is mined.
    */
   refresh?: number | 'never' | 'everyBlock'
+  /**
+   * Local storage overriding purposes
+   */
+  localStorageFunctions: LocalStorageFunctions
```

Comment:
Consider using `WindowLocalStorage['localStorage']` type instead of creating your own.